uninstall.php

```diff
@@ -11,4 +11,22 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() && 100 >= get_sites( array( 'count' => true ) ) ) {
+
+	$site_ids = get_sites(
+		array(
+			'fields'                 => 'ids',
+			'number'                 => '100',
```

Comment:
The number parameter defaults to 100. https://developer.wordpress.org/reference/classes/wp_site_query/__construct/ We should remove it.

Reply:
I think we should rather keep this here, since we really want it to be 100 sites. WordPress core's default could possibly change, so I think explicitly passing the 100 here makes sense.

However, it should probably be an integer per the documented values.
```suggestion
			'number'                 => 100,
```

Reply:
Make sense. Thanks for the clarification, Felix.